.gitignore

```diff
@@ -1,13 +1,14 @@
 Cargo.lock
 target
-.env
 *.db
-teleport.env
 exex.manifest.sgx
 exex.manifest
 exex.sig
 *.csr
 *.pem
 *.tmp
 certbot
-tokens.json
\ No newline at end of file
+tokens.json
+
+*.env
```

Comment:
teleport.env should not be ignored since it contains public data to bake into the enclave